docs/real-time-analytics/filter-logs-for-splunk/process.sql

```diff
@@ -0,0 +1,49 @@
+SET 'auto.offset.reset' = 'earliest';
+
+CREATE STREAM audit_log_events (
+            id VARCHAR, 
```

Comment:
@bbejeck would it be ok to make the preceding white space smaller, e.g. two spaces only?  (Here and throughout the ksqlDB statements)